src/test_l2_tvSettings.c

```diff
@@ -481,12 +475,12 @@ void test_l2_tvSettings_VerifyNoVideoSource(void)
 
     UT_LOG_DEBUG("Invoking GetCurrentVideoSource() with valid pointer");
     status = GetCurrentVideoSource(&currentSource);
-    UT_LOG_DEBUG("Return status: %d, Current Source: %d", status, currentSource);
+    UT_LOG_DEBUG("Return status: %d, Current Source: %d", status, (int)currentSource);
```

Comment:
tvVideoSrcType_t  is an enum, do we really require to cast with int type?

Reply:
Okay. I have updated as per your feedback